package.json

```diff
@@ -6,6 +6,12 @@
     "eslint": "^4.1.1",
     "react": "^15.6.1",
     "react-dom": "^15.6.1",
+    "react-flexbox-grid": "^1.1.3",
+    "react-fontawesome": "^1.6.1",
```

Comment:
Several questions... Do we need to add the whole font awesome library? How many font awesome fonts do we use on the site? Is the react version stable enough or are we better off with the original CSS library added through a cdn?